src/lib/orionTypes/EntityTypesResponse.cpp

```diff
@@ -66,11 +66,6 @@ std::string EntityTypesResponse::render(ConnectionInfo* ciP, const std::string&
   std::string out                 = "";
   std::string tag                 = "entityTypesResponse";
 
-  if ((ciP->uriParam["attributesFormat"] == "object") && (ciP->outFormat == JSON))
```

Comment:
Yeah, we should probably mention that we no longer support the attributeFormat URI param (but only for the '/v1/contextTypes' request, right?), and that it is not needed as this response is now always in json object format.